vyper/semantics/types/function.py

```diff
@@ -695,8 +709,8 @@ def _parse_return_type(funcdef: vy_ast.FunctionDef) -> Optional[VyperType]:
 
 
 def _parse_decorators(
-    funcdef: vy_ast.FunctionDef,
-) -> tuple[FunctionVisibility, StateMutability, bool]:
+    funcdef: vy_ast.FunctionDef, is_interface: bool = False
```

Comment:
not sure we need this `is_interface` kwarg

Reply:
lifted the interface-dependant checks in: https://github.com/vyperlang/vyper/pull/4178/commits/b8389641c6020b2b78728e4f78e6b663450601e1